src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsregisterstodeliusupdate/config/ResourceServerConfiguration.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsregisterstodeliusupdate.config
+
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
+import org.springframework.security.config.http.SessionCreationPolicy
+
+@Configuration
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
+class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
+  override fun configure(http: HttpSecurity) {
+    http
+      .sessionManagement()
+      .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+      .and().csrf().disable()
+      .authorizeRequests { auth ->
+        auth.antMatchers(
+          "/webjars/**", "/favicon.ico", "/csrf",
+          "/health/**", "/info", "/ping",
+          "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html"
```

Comment:
very minor, not sure we have a `ping` endpoint there days, isn't `health/ping` which is covered by `/health/**` ?

Reply:
yes - will remove in next PR